core/src/main/scala/com/clairvoyant/restonomer/core/app/RestonomerContext.scala

```diff
@@ -30,21 +30,21 @@ class RestonomerContext(val restonomerContextDirectoryPath: String) {
 
   val configVariables: Map[String, String] = loadConfigVariables(CONFIG_VARIABLES_FILE_PATH)
 
-  val checkpoints: List[CheckpointConfig] = loadConfigsFromDirectory[CheckpointConfig](
-    configDirectoryPath = CHECKPOINTS_CONFIG_DIRECTORY_PATH,
-    configVariablesSubstitutor = ConfigVariablesSubstitutor(configVariables = configVariables)
-  )
-
-  def runCheckpoint(checkpointName: String): Unit =
-    checkpoints
-      .find(_.name == checkpointName) match {
-      case Some(checkpointConfig) =>
-        runCheckpoint(checkpointConfig)
-      case None =>
-        throw new RestonomerContextException(s"The checkpoint: $checkpointName does not exists.")
-    }
+  def runCheckpoint(checkpointName: String): Unit = {
+    val filePath = s"$CHECKPOINTS_CONFIG_DIRECTORY_PATH/$checkpointName.conf"
+
+    val checkpoint: CheckpointConfig = loadConfigsFromFilePath[CheckpointConfig](
+      filePath,
+      configVariablesSubstitutor = ConfigVariablesSubstitutor(configVariables = configVariables)
```

Comment:
Its a good practice to be consistent while using named arguments. Either use it for all or do not use it at all.
Use named argument for filePath also.

Reply:
resolved